set-java.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# The selection used by update-alternatives for each java version
+if [ "$1" == "11" ]; then
+    # Java 11
+    selection="1"
+    runcommand="env JENKINS_OPTS=\"--enable-future-java\" ./run.sh firefox latest -DforkCount=1 -Dmaven.test.failure.ignore=true -B -Dtest=..."
```

Comment:
should not it also pass Java modules here? or is it handled elsewhere?

Reply:
Have not turned out to be needed at all.